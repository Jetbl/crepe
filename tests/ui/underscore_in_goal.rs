// This test tries to derive a rule with an ignored variable "_"

use crepe::crepe;

crepe! {
    struct Ok(i32);
    Ok(_) <- Ok(_);
}

fn main() {}
