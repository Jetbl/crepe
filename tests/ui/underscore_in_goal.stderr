error: Cannot have _ in goal atom of rule.
 --> $DIR/underscore_in_goal.rs:7:8
  |
7 |     Ok(_) <- Ok(_);
  |        ^
